core/iwasm/interpreter/wasm_loader.c

```diff
@@ -13576,6 +13581,11 @@ wasm_loader_prepare_bytecode(WASMModule *module, WASMFunction *func,
             {
                 uint32 opcode1;
 
+#if WASM_ENABLE_WAMR_COMPILER != 0
+                /* Mark the SIMD instruction is used in this module */
+                module->is_simd_used = true;
```

Comment:
Should we add more checks? e.g., whether a function type's param/result type is v128, whether a function's local type is v128, whether a global type is v128?